sdk/keyvault/azure-keyvault-certificates/azure/keyvault/certificates/_client.py

```diff
@@ -554,6 +554,11 @@ def list_deleted_certificates(self, **kwargs):
         """
         max_page_size = kwargs.pop("max_page_size", None)
 
+        if self.api_version == "2016-10-01" and kwargs.get("include_pending"):
```

Comment:
The main reason why I'm throwing here is because of the generated code based off of the swagger. Since it's not mentioned as a parameter, it never gets popped from kwargs, so eventually request is going to throw an error saying "i don't know what include_pending is". If we want to expose include_pending for 2016-10-01, we'd have to change the swagger